src/index.html

```diff
@@ -16,17 +16,46 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page">
+    <div class="search-bar">
+      <div
+        class="search-bar__item search-bar__item--big"
+        data-qa="big"
+      >
+        <div class="icon-big"></div>
```

Comment:
The `div` with the class `icon-big` is empty. If it's used for displaying an icon, consider using an `img` tag with an appropriate `alt` attribute for accessibility. If it's used for styling purposes, consider using CSS pseudo-elements like `::before` or `::after`.